alacritty_terminal/src/term/search.rs

```diff
@@ -434,6 +434,46 @@ impl<T> Term<T> {
 
         point
     }
+
+    /// Find the next shell prompt.
+    pub fn next_prompt(&self, mut point: Point) -> Option<Point> {
+        // If we're at the start of the prompt go to the next.
+        if point.column == Column(0) {
+            point.line += 1
+        } else {
+            point.column = Column(0);
+        }
```

Comment:
```suggestion
        point.column = Column(0);
```

A bit shorter like this and probably compiles down to the same thing anyway?